lumicks/pylake/file.py

```diff
@@ -209,3 +210,34 @@ def fdcurves(self) -> Dict[str, FDCurve]:
     @property
     def markers(self) -> Dict[str, Marker]:
         return self._get_object_dictionary("Marker", Marker)
+
+    def export_h5(self, filename, compression_level=5, omit_data={}):
```

Comment:
Maybe re-export instead of export? Since we aren't really exporting to a different format.
`reexport_h5()`?

Reply:
How about `save_as`? It's a fairly user-oriented name, and since `h5` is already implied we don't have to put it in the function name. If we ever decide to support more file types, we could always add a keyword argument with a default.

Reply:
`save_as()` 👍